controllers/drplacementcontrol_controller_test.go

```diff
@@ -185,8 +187,8 @@ func updateManagedClusterViewWithVRG(mcv *fndv2.ManagedClusterView, replicationS
 		TypeMeta:   metav1.TypeMeta{Kind: "VolumeReplicationGroup", APIVersion: "ramendr.openshift.io/v1alpha1"},
 		ObjectMeta: metav1.ObjectMeta{Name: DRPCName, Namespace: DRPCNamespaceName},
 		Spec: rmn.VolumeReplicationGroupSpec{
-			VolumeReplicationClass: "volume-rep-class",
```

Comment:
`ocm-minikube-ramen.sh` creates this object, but it seems it is no longer required, so pull request https://github.com/RamenDR/ramen/pull/166 removes the creation and deletion of it.